ports/zoe/portfile.cmake

```
@@ -2,8 +2,8 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO winsoft666/zoe
     HEAD_REF master
-    REF c4de5eef2139050dc6919170487bc5c5044c9fcf
-    SHA512 72e439b27405784878c1784fbac7308b021636231402e1fd88cfe3e850f0f539d726dc121278bbe3017aea613c6d4dff0a4034ad5488ebea06d681d81e4c8063
+    REF 51851d8deb9c8d61627c9bdbf1c33f3a8e458fda

```

```suggestion
    REF "v${VERSION}"
```
Please use `${VERSION}` instead of commit ID.